src/pages/Home/Process.js

```diff
@@ -37,63 +37,66 @@ const ProcessDescription = styled.div`
-    height: "225",
-    alt: "A watercolor of a hand pruning a fir bow",
+    width: '200',
+    height: '225',
+    alt: 'A watercolor of a hand pruning a fir bow',
   },
   {
-    processTitle: "Distillation",
+    processTitle: 'Distillation',
     description:
-      "Our 100% solar-powered still boils water and pushes steam through the plant material to capture the aromatic oils. The steam is recondensed into two products: pure essential oil, and hydrosol, a gentle botanical water.",
-    width: "1550",
+      'Our solar-powered still boils water and pushes steam through plant material to capture the aromatic oils. The steam cools into two products: pure essential oil, and hydrosol, a gentle botanical water. We bottle and sell these whole, or use them as ingredients in our other offerings, such as the scent for our 100% plant-based candles.',
+    width: '1550',
     img: `${distill}`,
     webp: `${distillWebp}`,
-    height: "549",
-    alt: "A watercolor of a copper still for distilling essential oils",
+    height: '549',
+    alt: 'A watercolor of a copper still for distilling essential oils',
   },
   {
-    processTitle: "Aging",
+    processTitle: 'Aging',
     description:
-      "Unlike many mass-market oils, we age our oils between 3 to 16 weeks. By aging, the oil can reach its true scent, the exact smell you would get from crushing the fresh plant in your hand.",
+      'We age our oils between 3 to 16 weeks. By aging, the oil can reach its true scent, the exact smell you would get from crushing the fresh plant in your hand.',
     img: `${aging}`,
     webp: `${agingWebp}`,
-    width: "1950",
-    height: "700",
-    alt: "A watercolor of differnt moon phases",
+    width: '1950',
+    height: '700',
+    alt: 'A watercolor of differnt moon phases',
   },
   {
-    processTitle: "Bottle",
+    processTitle: 'Bottle',
     description:
-      "At last, our oils are hand bottled and ready for sale. You can find us at the Bayview Farmers Market, the lovely stores listed below, or on your doorstep. Online orders are sent in 100% recycled, plastic-free, guilt-free packaging.",
+      'At last, our oils are hand bottled and ready for sale. You can find us at the Bayview Farmers Market, the lovely stores listed below, or on your doorstep by ordering right here on our website. Online orders are sent in 100% recycled, plastic-free, guilt-free packaging and with gratitude from your humble makers.',
     img: `${bottle}`,
     webp: `${bottleWebp}`,
-    width: "200",
-    height: "225",
-    alt: "A watercolor of a flower",
+    width: '200',
+    height: '225',
+    alt: 'A watercolor of a flower',
   },
 ];
 
 const Process = () => {
   return (
-    <ComponentWrapper id="process">
+    <ComponentWrapper id='process'>
       <ProcessDescription>
         <StyledH2>The Process</StyledH2>
-        <p>From start to finish, oils that are good to mother earth.</p>
+        <p>
+          We grow the plants, we make the goods – <br />
```

Comment:
This `<br />` is making things look a bit funny at 375 BP, is this intentional?
![Screen Shot 2022-05-15 at 2 44 52 PM](https://user-images.githubusercontent.com/49624436/168495297-06691138-4742-4175-8202-c2faec549527.png)

Reply:
No, not intentionally, I think I have a fix though